geth/node/manager.go

```diff
@@ -200,6 +200,32 @@ func (m *NodeManager) populateStaticPeers() error {
 	return nil
 }
 
+func (m *NodeManager) removeStaticPeers() error {
+	if !m.config.BootClusterConfig.Enabled {
+		log.Info("Boot cluster is disabled")
+		return nil
+	}
+	for _, enode := range m.config.BootClusterConfig.BootNodes {
+		err := m.delPeer(enode)
+		if err != nil {
+			log.Warn("Boot node deletion failed", "error", err)
```

Comment:
I would return an error in such a case. Also, the interface of this method suggests that it is possible to return an error but it never happens.

And the final thing, as this method says `removeStaticPeers` we should be consistent and log `Static peer deletion failed`.